astropy/nddata/tests/test_utils.py

```diff
@@ -167,7 +167,9 @@ def test_extract_array_even_shape_rounding():
     positions = (-0.99, -0.51, -0.5, -0.49, -0.01, 0)
     exp1 = (-99, 0)
     exp2 = (0, 1)
-    expected = [exp1,] * 6 + [
```

Comment:
Like done below, this should go on one line by removing the trailing `,`, which are not needed for lists.